packages/rancher-monitoring/generated-changes/patch/templates/grafana/dashboards-1.14/kubelet.yaml.patch

```diff
@@ -9,3 +15,275 @@
    name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" $) "kubelet" | trunc 63 | trimSuffix "-" }}
    annotations:
  {{ toYaml .Values.grafana.sidecar.dashboards.annotations | indent 4 }}
+@@ -107,7 +108,7 @@
+                         "tableColumn": "",
+                         "targets": [
+                             {
+-                                "expr": "sum(up{cluster=\"$cluster\", job=\"kubelet\", metrics_path=\"/metrics\"})",
++                                "expr": "sum(up{cluster=\"$cluster\", job=\"{{ include "exporter.kubelet.jobName" . }}\", metrics_path=\"/metrics\"})",
```

Comment:
metrics_path should come from the helper function throughout this file.

Reply:
Metrics path never changes here; this is the upstream value.  The only thing we're changing is the job name

Reply:
Ah sorry. I meant to delete this comment before submitting the review.